src/IBusBM.cpp

```diff
@@ -102,13 +102,16 @@ void IBusBM::begin(HardwareSerial &serial, int8_t timerid, int8_t rxPin, int8_t
   this->len = 0;
   this->chksum = 0;
   this->lchksum = 0;
+  for (uint8_t i = 0; i < PROTOCOL_CHANNELS; i++) {
+    channel[i] = 1500;
+  }
 
   // we need to process the iBUS sensor protocol handler frequently enough (at least once each ms) to ensure the response data
   // from the sensor is sent on time to the receiver
   // if timerid==IBUSBM_NOTIMER the user is responsible for calling the loop function
   this->IBusBMnext = IBusBMfirst;
 
-  if (!IBusBMfirst && timerid != IBUSBM_NOTIMER) {
+  if (!IBusBMfirst && timerid != (void *)IBUSBM_NOTIMER) {
```

Comment:
This fixed a compiler warning.

Reply:
Thank you! I'd been meaning to get around to fixing that, but hadn't taken the time to.